petastorm/tests/test_spark_dataset_converter.py

```diff
@@ -24,12 +24,14 @@
                                StringType, StructField, StructType)
 from six.moves.urllib.parse import urlparse
 
-from petastorm.spark.spark_dataset_converter import make_spark_converter
+from petastorm.spark.spark_dataset_converter import enable_delta, \
+    make_spark_converter
 
 
 class TfConverterTest(unittest.TestCase):
 
     def setUp(self):
+        enable_delta()
```

Comment:
We should make two sub-class for `TfConverterTest`:

* `TfConverterTestOnDelta(TfConverterTest)`: in setup call `enable_delta` and then `assert _is_delta_available()` 

* `TfConverterTestOnParquet(TfConverterTest)`  in setup call `assert not _is_delta_available()`

Reply:
If `TfConverterTest` does not inherit `unittest.TestCase`, the `test_...` functions cannot use assert methods like `self.assertEqual()`.